gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineExactDependencies.java

```diff
@@ -278,7 +282,7 @@ private static String asDependencyString(ResolvedArtifact artifact, boolean with
                     .append(projectComponentId.getProjectPath())
                     .append("')");
             if (withName) {
-                builder.append(" (").append(artifact.getId().getDisplayName()).append(")");
+                builder.append(" <-- ").append(artifact.getName());
```

Comment:
Drive by to make this out a bit more useful. The current output is verbose and has confused users.

**Before**

```
Execution failed for task ':foo:checkUnusedDependenciesMain'.
> Found 1 dependencies unused during compilation, please delete them from 'foo/build.gradle' or choose one of the suggested fixes:
        project(':bar') (main (project :bar))
```

**After**

```
Execution failed for task ':foo:checkUnusedDependenciesMain'.
> Found 1 dependencies unused during compilation, please delete them from 'foo/build.gradle' or choose one of the suggested fixes:
        project(':bar') <-- main
```